ua-snap/arctic-eds
# Adds Hydrology section to the ArcticEDS report

This PR adds the Hydrology section of the ArcticEDS report.

The section contains a summary of era-based annual min-mean-max for three models and two scenarios. The values in these summary tables can be changed between imperial and metric units as well. 

The section also contains the CSV preview and the ability to download a CSV for the given location. 

Currently, the GeoNetwork endpoint for this data is not available, but we can update where the data is pointed as soon as it is available. 

Finally, it has the academic references to this work per the data API endpoint for hydrology.

For testing, be sure to have the [hydrology_eds](https://github.com/ua-snap/data-api/tree/hydrology_eds) branch of the data API locally running and set your environment variables as such:

`export SNAP_API_URL=http://localhost:5000`
`export RASDAMAN_URL=https://apollo.snap.uaf.edu/rasdaman/ows`

Things to test:

* Ensure that the hydrology section shows up using the mock.json 
* Ensure that the values start out as imperial units and convert properly to metric and then to imperial again
* Turn off the mock API call and ensure that a request to the local SNAP API returns hydrology data
* Check that the radio buttons change the data in the summary tables for runoff, evapotranspiration, and soil moisture.

Closes #339
